app/src/organisms/LabwarePositionCheck/ResultsSummary.tsx

```diff
@@ -168,6 +174,9 @@ export const ResultsSummary = (
           alignSelf={ALIGN_FLEX_END}
           onClick={() => handleApplyOffsets(offsetsToApply)}
           buttonText={i18n.format(t('apply_offsets'), 'capitalize')}
+          iconName={isSubmittingAndClosing ? 'ot-spinner' : null}
+          iconPlacement={isSubmittingAndClosing ? 'startIcon' : null}
```

Comment:
maybe you can do this to avoid typing the props as null:

```suggestion
          iconName={isSubmittingAndClosing && 'ot-spinner'}
          iconPlacement={isSubmittingAndClosing && 'startIcon'}
```

Reply:
hmm that won't either because `isSubmittingAndClosing && 'ot-spinner'` is a logical expression that will either return `ot-spinner` (valid) or `false` (not valid)